lib/parse-args.js

```diff
@@ -0,0 +1,33 @@
+'use strict';
+
+const minimist = require('minimist');
+
+const parse = (args) => {
+  if (!args || args.length === 0) {
+    throw new Error('No src folder or pattern specified');
+  }
+  const parsed = minimist(args);
+  const writeShared = parsed.writeShared || parsed.w;
+
+  let sources = parsed._;
+  let shared = parsed.s || parsed.shared;
+
+  if (shared && !Array.isArray(shared)) {
+    shared = [shared];
+  }
```

Comment:
no, because shared is what's being used, `[].concat(shared)` will be a new array. Unless you mean remove the logic and always `concat`, in which case we'll get [arr, arr]